Documentation/operations/upgrade.rst

```diff
@@ -320,10 +320,19 @@ Helm Options
 * ``clustermesh.apiserver.tls.ca.cert`` has been deprecated in favor of ``tls.ca.key``, and will be removed in 1.13.
 * ``clustermesh.apiserver.tls.ca.key`` has been deprecated in favor of ``tls.ca.key``, and will be removed in 1.13.
 * ``tls.enabled`` has been removed as this attribute is not used at all.
+* ``tcFilterPriority`` has been added. See description of Cilium's ``tc-filter-priority`` option in New Options, below.
```

Comment:
As said below, I'd remove that from Helm, the doc, and mark the agent flag as hidden.

Reply:
Worth noting, this will still be configurable via Helm, you would just need to use extraArgs.

Reply:
Good point, given this is very niche, extraArgs should suffice here.